### DISCUSSION TITLE
Some features possible with shizuku?

### DISCUSSION THREAD

**Discussion by terminal-cs** [Mahmud0808/Iconify#187] [Ideas]
Hello, this project is great, however I cannot have root until the device is fully paid off. I was wondering if it's possible to leverage the power of shizuku to use some of the modifications this app has without requiring root. Obviously not all would be supported, however I feel it may be possible for some tweaking using shizuku.

**Comment by ThatOneCalculator** [comment]
Agreed, especially as someone who uses GrapheneOS. A smaller subset of Shizuku-compatible features would be great!

**Comment by Mahmud0808** [comment]
Currently not possible. Module is mandatory as it keeps everything inside the module.
